src/ngraph/op/util/broadcasting.hpp

```diff
@@ -53,27 +55,6 @@ namespace ngraph
         /// \return Broadcast output.
         std::shared_ptr<Node> numpy_style_broadcast(const Output<Node>& value, const Shape& shape);
 
-        /// \brief Cast shape of two outputs to make them compatible for an element-wise binary
-        ///        operation.
-        ///
-        /// If necessary the right-hand-side argument will be broadcast to match the shape
-        /// of left-hand-side argument. The starting of the mutually equal shape is
-        /// specified by the argument "start_match_axis", and if it is not set,
-        /// suffix matching is assumed.
-        ///
-        /// This style of broadcast was used in ONNX Op sets prior to version 7, where it was
-        /// replaced by numpy-style broadcasting.
-        ///
-        /// \param left Node which contain input of binary op.
-        /// \param right Node which contain input of binary op.
-        /// \param start_match_axis position in shape denoting start of the mutually equal shape
-        ///
-        /// \return Left and right node after broadcasting.
-        NodeVector legacy_style_broadcast_for_binary_operation(const std::shared_ptr<Node>& left,
```

Comment:
Could you rename `legacy_style_broadcast_values_for_binary_operation` -> `legacy_style_broadcast_for_binary_operation`?
Since `legacy_style_broadcast_for_binary_operation` has been removed there is a good opportunity to remove `values` from function name.
Especially when opset1 version of `legacy_style_broadcast_values_for_binary_operation` is named `legacy_style_broadcast_for_binary_operation`.

Reply:
Done